### PR TITLE
MISUV-5083 : Create Separate Login Page - Agent Login - FE

### PR DESCRIPTION
**Pull request by AshokGitAccount**

### CODE REVIEW

**Review by avelx — APPROVED** [review]
approved

**Comment by AshokGitAccount** [comment]
@avelx Please review the last commit and merge the PR if there is no change.

File: test/views/agent/HomePageViewSpec.scala [hunk 1]
```diff
@@ -123,9 +123,10 @@ class HomePageViewSpec extends TestSupport with FeatureSwitching with ViewSpec {
         document.select("h1").text() shouldBe messages("home.agent.heading")
       }
 
-      s"have the hint with the users name '$testUserName' and utr '$testSaUtr' " in new Setup {
+      // Commented the test case as the property in view is also commented
```
Comment (avelx) [inline test/views/agent/HomePageViewSpec.scala:126]: why this file was changed at all ? we are not doing any changes in the home page

File: test/views/HomePageViewSpec.scala [hunk 2]
```diff
@@ -153,7 +153,7 @@ class HomePageViewSpec extends TestSupport {
     }
 
     s"have the subheading with the users name '$testUserName'" in new Setup {
-      getTextOfElementById("sub-heading") shouldBe Some(testUserName)
+      getTextOfElementById("sub-heading") shouldBe None
```
Comment (avelx) [inline test/views/HomePageViewSpec.scala:156]: this story is not related to any changes in the home controller or spec

File: app/views/Home.scala.html [hunk 3]
```diff
@@ -141,7 +141,10 @@
                         </span>
                     }
                     <span class="govuk-visually-hidden">@messages("home.agent.visualHidden.clientsName")</span>
+                @*
```
Comment (avelx) [inline app/views/Home.scala.html:144]: pls remove this, I was referring to the fact that we need to find out where about this data  commit from; 
revert last changes related to userName bug, this to be address in the separate PR

Comment (AshokGitAccount) [reply app/views/Home.scala.html]: reverted

**Review by avelx — APPROVED** [review]
